src/apps/statements/expressPresenter/utils/getStatements/getSingleStatement.ts

```diff
@@ -22,7 +22,13 @@ export default async (opts: Options) => {
   const timestamp = new Date().toISOString();
   const resultOpts = getStatementsResultOptions(queryParams, client);
 
-  checkUnknownParams(queryParams, ['statementId', 'voidedStatementId', 'format', 'attachments']);
+  checkUnknownParams(queryParams, [
+    'statementId',
+    'voidedStatementId',
+    'format',
+    'attachments',
+    'readonly',
```

Comment:
Adding a validator like `/Users/lydiaross/dev/xapi-service/src/apps/statements/expressPresenter/utils/validateStatementProcessingPriority.ts` was considered to check that the value is 0, but given that we do nothing with this parameter, I think it's unnecessary